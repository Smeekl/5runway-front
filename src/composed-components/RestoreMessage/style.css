.restore-message {
  padding: 62px 35px 0 35px;
}

.restore-message__title {
  width: 200px;
  font-weight: 700;
  font-size: 24px;
  color: var(--color-primary);
}

.restore-message__description {
  font-weight: 300;
  margin-bottom: 32px;
}

.restore-message__btn {
  background: var(--color-primary-blue);
  border-radius: 15px;
  width: 100%;
  height: 48px;
  border: none;
}

.restore-message__btn > span {
  color: var(--color-secondary-white);
}

.restore-message__btn:hover,
.restore-message__btn:focus {
  background: var(--color-primary);
}
