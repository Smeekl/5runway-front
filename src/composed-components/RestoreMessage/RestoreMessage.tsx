import './style.css';

const RestoreMessage = () => (
  <div className="restore-message">
    <h3 className="restore-message__title">Message sent successfully</h3>
    <p className="restore-message__description">
      If you input correct email and do not receive the message in your email,
      please check your spam folder.
    </p>
  </div>
);

export default RestoreMessage;
