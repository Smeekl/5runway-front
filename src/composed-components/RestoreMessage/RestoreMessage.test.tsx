import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import RestoreMessage from './RestoreMessage';

describe('RestoreMessage UI tests', () => {
  test('should render correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <RestoreMessage />
      </MemoryRouter>
    );

    expect(container).toMatchSnapshot();
  });

  test('should be in the document', () => {
    const { container } = render(
      <MemoryRouter>
        <RestoreMessage />
      </MemoryRouter>
    );

    expect(container).toBeInTheDocument();
  });
});
