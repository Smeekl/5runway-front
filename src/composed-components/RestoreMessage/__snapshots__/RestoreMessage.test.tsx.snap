// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestoreMessage UI tests should render correctly 1`] = `
<div>
  <div
    class="restore-message"
  >
    <h3
      class="restore-message__title"
    >
      Message sent successfully
    </h3>
    <p
      class="restore-message__description"
    >
      If you input correct email and do not receive the message in your email, please check your spam folder.
    </p>
  </div>
</div>
`;
